deploy/dex_structure.py

```diff
@@ -1512,6 +1537,9 @@ def farm_boosted_deploy(self, contracts_index: str, deployer_account: Account, n
                                                         lock_epochs)
             if not network_providers.check_simple_tx_status(tx_hash, "set lock epochs in farm"): return
 
+            deployed_contract.resume(deployer_account, network_providers.proxy)
```

Comment:
`tx_hash = deployed_contract.resume(deployer_account, network_providers.proxy)`
otherwise you will check below the previous tx_hash

Reply:
removed completely since the need to resume was removed from contract